pkgs/io_file/lib/src/file_system.dart

```diff
@@ -30,4 +54,13 @@ base class FileSystem {
   Uint8List readAsBytes(String path) {
     throw UnsupportedError('readAsBytes');
   }
+
+  /// Write the given bytes to a file.
+  void writeAsBytes(
```

Comment:
It has bothered dart:io calls this "write as bytes" to me it should be "write from bytes" or just "write bytes" `writeBytes`. I guess it is to mirror `readAsBytes` - but still...

Not sure we want to change it for this library - but just a thought...